unit_tests/test_lib_charms_ovn_charm.py

```diff
@@ -61,7 +61,7 @@ def test_dpdk_device(self):
         self.assertEquals(self.target.dpdk_device.driver, 'fakedriver')
 
     def test_sriov_device(self):
-        self.assertDictEqual(self.target.sriov_device,
+        self.assertDictEqual(self.target.sriov_device(),
```

Comment:
This is testing the legacy behavior. I guess we could have one more such tiny test covering the new `get_map()`?

Reply:
So, testing the innards of the SRIOV context belongs to the unit tests in charm helpers. If anything we should probably remove this test here.

Reply:
Yes I thought about that, on the other hand this kind of tests is good to catch non-backward-compatible changes done to charmhelpers. You could imagine someone changing the behavior of the called function and changing the corresponding test in charmhelpers, so that it's passing there in charmhelpers and lands. It would then turn red here and make obvious that something has to be done in this layer.

I don't mind too much though, as you like.

Reply:
The test mocks the context in its entirety, so all it tests is that when you access the property you get a object that comes from a given location. The return value is defined in the test itself.

Reply:
ah true :zipper_mouth_face: